guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,268 @@
+
+**6. Bitcoin Transfer** - Once the tokens are verified and destroyed, the provider sends an equivalent amount of bitcoin to the user. 
+
+#### Terminology Translation Table
+
+| Action | Cashu Term | Fedimint Term |
+|-------|----------|----------|
+| Create ecash | Mint | Issue |
+| Send ecash | Swap | Reissue | 
+| Convert ecash to bitcoin | Melt | Redeem |
+
+### Sending and Receiving Ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   alt-text = "An illustration of the process of sending and receiving ecash tokens to various users"
+   caption = "Sending and receiving ecash tokens."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+When a user sends an ecash token to another user, the wallet verifies the mint's signature to confirm the token's authenticity. The mint then checks its database to ensure the token has not been previously spent. If the token is valid, it is destroyed and replaced with a new token, which is then issued to the recipient. This cycle of destroying the old token and issuing a new one prevents double spending while maintaining user privacy and security.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@1x.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@2x.jpg"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@2x.jpg"
+   alt-text = "An illustration comparing how on-chain bitcoin, lightning, and ecash handle the sending of bitcoin."
+   caption = ""
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+Sending ecash and receiving can be done via multiple methods to suit different transaction needs. One of the advantages of bitcoin-backed ecash is that the tokens are just strings of text, which are very flexible and can be transmitted via any medium or rail. Some unique ways that ecash can be sent and received, which are not possible with on-chain or Lightning transactions, include:
+
+#### Text Message, Such as Email or SMS:
+Users can send ecash by copying and pasting the token string. This is particularly useful for online transactions or when using text based communication platforms.
+
+#### Bluetooth Transmission:
+Most applications allow users to share a token via Bluetooth. However, more comprehensive Bluetooth sending and receiving functionalities have not yet been fully developed.
```

Comment:
A way to send the token so that a mobile phone can recognize it as an ecash token and know what to do with it, such as opening an application to redeem it. Some apps allow you to “share” an ecash token via Bluetooth, but the receiving phone typically only recognizes it as a text string, which often results in the token being opened in the phone’s notes application instead of the intended app.

Would replacing comprehensive with a "full end to end" be better? Not sure how to communicate this without making it too wordy.